ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -65,14 +66,43 @@ export const ApplicationSummary = (props: {app: models.Application; updateApp: (
             edit: (formApi: FormApi) => (
                 <DataLoader
                     load={() =>
-                        services.clusters.list().then(clusters =>
-                            clusters.map(cluster => ({
-                                title: clusterTitle(cluster),
-                                value: isDestName ? cluster.name : cluster.server
-                            }))
-                        )
+                        Promise.all([
+                            services.clusters.list().then(clusters => clusters.map(cluster => cluster.server).sort()),
+                            services.clusters.list().then(clusters => clusters.map(cluster => cluster.name).sort())
+                        ]).then(([clusterURLs, clusterNames]) => ({clusterURLs, clusterNames}))
                     }>
-                    {clusters => <FormField formApi={formApi} field={fieldDest} componentProps={{options: clusters}} component={FormSelect} />}
+                    {({clusterURLs, clusterNames}) => {
+                        return (
+                            <div className='row'>
+                                {(destFormat.toUpperCase() === 'URL' && (
+                                    <div className='columns small-10'>
```

Comment:
I've tried to update an existing application and use cluster name instead of server . When I've tried to save application the cluster name was set to server field, instead of name.